### PR TITLE
fix: hpa and variables

### LINKED ISSUE #49: [BUG]: Fix hpa and file name consistency
### Description of the bug

Misconfiguration of hpa and some other variables  

### Steps To Reproduce

Enabling hpa 

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by mfroembgen**
Closes #49 

## ✅ Checks

- [x] My pull request adheres to the code style of this project
- [x] My code requires changes to the documentation
- [x] I have updated the documentation as required
- [x] All the tests have passed

### CODE REVIEW

**Review by talboren — APPROVED** [review]
LGTM

@shahargl wdyt?

File: charts/keep/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # keep
 
-![Version: 0.0.5](https://img.shields.io/badge/Version-0.0.5-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.2.1](https://img.shields.io/badge/AppVersion-0.2.1-informational?style=flat-square)
+![Version: 0.1.17](https://img.shields.io/badge/Version-0.1.17-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.25.2](https://img.shields.io/badge/AppVersion-0.25.2-informational?style=flat-square)
```
Comment (shahargl) [inline charts/keep/README.md:3]: is there a way somehow to use the value from `charts.yaml` so we won't need to update two files?

Comment (mfroembgen) [reply charts/keep/README.md]: I don't think so as it is autogenerated from the helm-docs pre-commit hook and need to be generated again on every version change.

**Comment by shahargl** [comment]
thanks!

**Pull request merged**